src/main/java/org/xolstice/maven/plugin/protobuf/ProtocCompileJavaScriptMojo.java

```diff
@@ -52,6 +52,7 @@
 
     /**
      * Additional comma-separated options to be passed to the JavaScript generator.
+     * E.g. import_style=commonjs,binary
```

Comment:
Can you please enclose the example options in `<tt>..</tt>` tags and also add a `@see <a href=...` javadoc link to https://github.com/protocolbuffers/protobuf/blob/master/js/README.md for further details?
Also please shift the example one line down, so that the warning comes first.

Reply:
Sure, I may have a few other additions which I think may be helpful.